app/views/users/show.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Here is the profile of a given user</h1>
\ No newline at end of file
+<%= render @user %>
+<% if @user.bio.present? %>
+	<div class="mt-5 notification">
+	  <h5 class="title is-6">Bio</h5>
+	  <p class="is-6"><%= @user.bio %></p>
+	</div>
+<% end %>
+<%= render partial: "posts/post", collection: @posts %>
```

Comment:
- [ ] As part of the requirements, the list of the 3 most recent post are to be displayed here. Your current implementation doesn't have that. I suggest that you fix it.

Your Design | Template Design
--|--
![image](https://user-images.githubusercontent.com/29071984/197719623-3730db00-9ee7-47dd-9fe1-ec3bdad1be31.png) | ![image](https://user-images.githubusercontent.com/29071984/197719439-6c26d33f-40a6-4268-8997-fc00bd30dd4d.png)